src/UraniumUI.Material/Controls/TreeViewNodeHolderView.cs

```diff
@@ -95,30 +94,11 @@ public TreeViewNodeHolderView(DataTemplate dataTemplate, TreeView treeView, Bind
         };
 
         this.Add(button);
-        this.Add(nodeChildren);
-
-        if (!string.IsNullOrEmpty(TreeView.IsExpandedPropertyName))
-        {
-            this.SetBinding(IsExpandedProperty, new Binding(TreeView.IsExpandedPropertyName, BindingMode.TwoWay));
-        }
-
-        if (!string.IsNullOrEmpty(TreeView.IsLeafPropertyName))
+        if (childrenBinding is Binding binding && !string.IsNullOrEmpty(binding.Path) && TryGetInitialChildren(binding.Path, out var children) && children?.Cast<object>().Any() == true)
```

Comment:
- TryGetInitialChildren(binding.Path, out var children) returns false if and only if children is not null.
- thus, you can simply write `children.Cast<object>().Any()` instead of `children?.Cast<object>().Any() == true`